readme.md

```diff
@@ -62,10 +62,26 @@ All new source files _must_ be in TypeScript, with useful type information for a
 ## To create a production-ready build of the SDK with source-maps side-by-side:
 
 `npm run build-with-sourcemaps` - the output will end up in `dist/*.js`. Use this if you want to host a version with source maps for easier debugging.
+
 ## To create a development build of the SDK with full source-maps, and monitor for changes:
 
 `npm run watch` - the output will end up in `dist/bundle.js` and will be rebuilt whenever any related files are changed.
 
+## To publish SDK to NPM repo and CDN
+
+1. Bump update version in `package.json` and then run:
+   ```bash
+   npm install
+   ```
+2. Use `Publish SDK to NPM and CDN` Action.
+
+### Prerelease
+
+If you intend to publish a prerelease version:
+
+1. Update the version number in package.json with the desired prerelease tag. For instance: `3.0.3-alpha.0`
```

Comment:
And then run `npm install`?

Reply:
`npm install` will update the `package-lock.json`, I think we should better not manually update it